app/javascript/src/utilities/ajax.js

```diff
@@ -2,22 +2,25 @@
 
 export type Method = 'GET' | 'POST' | 'DELETE'
 
-const ajax = (url: string, method: Method, body?: URLSearchParams): Promise<Response> => {
+type FetchFunction = (url: string, method: Method, body?: URLSearchParams) => Promise<Response>
```

Comment:
Why body is always `URLSearchParams` ?

Reply:
Because that's what the API endpoints expect, same as method being only 3 verbs. We will extend these types as we need. This module is not supposed to be an all-purpose solution but rather a handy helper for the few requests we do from React.

Reply:
ok